CONTRIBUTING.md

```diff
@@ -122,10 +122,10 @@ Please make sure that you check the items applicable to your pull request:
   * [ ] If changes were done to Autotools build, were they added to CMake and vice versa?
   * [ ] Is the pull request applicable to any other branches? If yes, which ones? Please document it in the GitHub issue.
   * [ ] Is the new code sufficiently documented for future maintenance?
-  * [ ] Does the new feature require a change to an existing API? See "API Compatibility Macros" document (https://hdfgroup.github.io/hdf5/develop/api-compat-macros.html)
```

Comment:
Does the current link checker action not check these? I would've thought it would catch these in other PRs.

Reply:
Yes the linkchecker catches these - but until everthing passes I couldn't let it fail the CI.